### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by kelvinqian00**

### CODE REVIEW

File: setup-env/action.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        distribution: ${{ inputs.java-distribution }}
+        java-version: ${{ inputs.java-version }}
+
+    - name: Setup Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ inputs.node-version }}
+
+    - name: Setup Clojure
+      uses: DeLaGuardo/setup-clojure@9.5
```
Comment (milt) [inline setup-env/action.yml]: this has advanced a few versions, don't know if you wanna update or not but I believe the latest is 10.2

**Review by milt — APPROVED** [review]
Looks good, see note about setup-clojure action version.

**Comment by kelvinqian00** [comment]
@milt I realized that we no longer need the inner `setup-env` folder. Also bumped the setup-clojure version number.

**Review by milt — APPROVED** [review]
LGTM

**Pull request merged**